packages-ts/starknet-gauntlet/src/wallet/ledgerWallet.ts

```diff
@@ -0,0 +1,94 @@
+import { IStarknetWallet } from './'
+import { SignerInterface, encode } from 'starknet'
+import { Abi, Invocation, InvocationsSignerDetails } from 'starknet/types'
+import { TypedData, getMessageHash } from 'starknet/utils/typedData'
+import { fromCallsToExecuteCalldataWithNonce } from 'starknet/utils/transaction'
+import { calculcateTransactionHash, getSelectorFromName } from 'starknet/utils/hash'
+import Stark from '@ledgerhq/hw-app-starknet'
+import { LedgerError } from '@ledgerhq/hw-app-starknet'
+import TransportNodeHid from '@ledgerhq/hw-transport-node-hid'
+
+// EIP-2645 path
+const PATH = "m/2645'/579218131'/0'/0'"
+
+function toHexString(byteArray: Uint8Array): string {
+  return Array.from(byteArray, function (byte) {
+    return `0${byte.toString(16)}`.slice(-2)
+  }).join('')
+}
+
+class LedgerSigner implements SignerInterface {
+  protected ledgerConnector: Stark
+
+  private constructor(lc: Stark) {
+    this.ledgerConnector = lc
+  }
+
+  static create = async () => {
```

Comment:
`path` should be accepted here. If undefined, it will use the default
```suggestion
  static create = async (path?: string) => {
```